src/Modules/Build.lua

```diff
@@ -1398,7 +1408,8 @@ function buildMode:OpenSaveAsPopup()
 		end
 	end
 	controls.label = new("LabelControl", nil, 0, 20, 0, 16, "^7Enter new build name:")
-	controls.edit = new("EditControl", nil, 0, 40, 450, 20, self.dbFileName and self.buildName, nil, "\\/:%*%?\"<>|%c", 100, function(buf)
+	controls.edit = new("EditControl", nil, 0, 40, 450, 20,
+	(self.dbFileName or self.buildName):gsub("[\\/:%*%?\"<>|%c]", "-"), nil, "\\/:%*%?\"<>|%c", 100, function(buf)
```

Comment:
This causes issues when using "Save As" on a local build:
![image](https://github.com/user-attachments/assets/6471a593-cf1a-4c5d-a434-61f1c2fcd198)

Reply:
Fixed.